pkgs/applications/blockchains/monero/default.nix

```diff
@@ -5,16 +5,11 @@
 , randomx, rapidjson
 , CoreData, IOKit, PCSC
 , trezorSupport ? true
-,   libusb1  ? null
-,   protobuf ? null
-,   python3  ? null
+, libusb1
+, protobuf
+, python3
```

Comment:
Maybe these can become a single line to indicate these are optional dependecies for Trezor support?

```
, trezorSupport ? true, libusb1, protobuf, python3
```